docs/source/datasets/index.rst

```diff
@@ -34,88 +35,140 @@ an individual simulant, with the columns representing different simulant attribu
+     - 
    * - Physical address ZIP code
      - :code:`zipcode`
+     - 
    * - Relationship to person 1 (head of household)
      - :code:`relationship_to_household_head` 
-   * - Sex (binary; 'male' or 'female')
+     - 'Person 1', 'head of household', and 'Reference person' are all synonymous in this context. Possible values for this indicator include:
+       Reference person; Biological child; Adopted child; Stepchild; Sibling; Parent; Grandchild; Parent-in-law; Child-in-law; Other relative;
+       Roommate; Foster child; and Other nonrelative.
+   * - Sex 
      - :code:`sex`  
+     - Binary; 'male' or 'female'.
    * - Race/ethnicity
      - :code:`race_ethnicity` 
+     - The exhaustive and mutually exclusive categories for the single composite 'race/ethnicity' indicator are as follows:
+       White; Black; Latino; American Indian and Alaskan Native (AIAN); Asian; Native Hawaiian and Other Pacific Islander (NHOPI); and
+       Multiracial or Some Other Race. 
 
 Household Surveys: ACS and CPS
 ------------------------------
 There are two simulated household survey datasets that can be used: the American
-Communities Survey (ACS) and the Current Population Survey (CPS). 
+Community Survey (ACS) and the Current Population Survey (CPS). 
+
+ACS is an ongoing household survey conducted by the US Census Bureau that gathers information on a rolling basis about
+American community populations. Information collected includes ancestry, citizenship, education, income, language proficienccy, migration, 
+employment, disability, and housing characteristics. To find out more about ACS, please visit the `ACS homepage <https://www.census.gov/programs-surveys/acs/about.html>`_.
+
+CPS is another household survey conducted by the US Census Bureau and the US Bureau of Labor Statistics. This survey is administered by Census 
+Bureau field representatives across the country through both personal and telephone interviews. CPS collects labor force data, such as annual
+work activity and income, veteran status, school enrollment, contingent employment, worker displacement, job tenure, and more. To find out more
+about CPS, please visit the `CPS homepage <https://www.census.gov/programs-surveys/cps.html>`_. 
 
 .. list-table:: **Simulant attributes**
    :header-rows: 1
 
    * - Attribute Name
      - Column Name
+     - Notes
    * - Unique simulant ID
-     - 
+     - simulant_id
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
    * - Household ID 
-     -  
+     - :code:`household_id` 
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking.
    * - First name
      - :code:`first_name`
+     - 
    * - Middle initial
      - :code:`middle_initial`
+     - 
    * - Last name
      - :code:`last_name`
+     - 
    * - Age
      - :code:`age`  
+     - Rounded down to an integer.
    * - Date of birth
      - :code:`date_of_birth`
+     - Formatted as MM/DD/YYYY.
```

Comment:
we better have a review of all date formats when the code freeze happens on Monday.